### PR TITLE
Add windows build badge

### PR DESCRIPTION
**Pull request by winterfroststrom**
Based on https://github.com/google/protobuf/blob/master/README.md and https://github.com/tensorflow/tensorflow/blob/master/README.md

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/426)

### CODE REVIEW

**Comment by xpconanfan** [comment]
Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---

*[README.md, line 3 at r1](https://beta.reviewable.io/reviews/google/mobly/426#-L9S3qg3CFfJLdvfspYV:-L9S3qg3CFfJLdvfspYW:b6xvyek) ([raw file](https://github.com/google/mobly/blob/124f0b4a1d101d62c5d5cca2a49c796cf306b3fd/README.md#L3)):*
> ```Markdown
> # Welcome to Mobly
> 
> | **`Linux`** | **`Windows`** |
> ```

Can we simply have a list instead of table?
Or better yet, change the badge text to directly say platform info so no extra text is needed

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/google/mobly/426)*

**Comment by winterfroststrom** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[README.md, line 3 at r1](https://beta.reviewable.io/reviews/google/mobly/426#-L9S3qg3CFfJLdvfspYV:-L9gINua05jpcHo3GVz3:bw3v6kl) ([raw file](https://github.com/google/mobly/blob/124f0b4a1d101d62c5d5cca2a49c796cf306b3fd/README.md#L3)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

Can we simply have a list instead of table?
Or better yet, change the badge text to directly say platform info so no extra text is needed
</blockquote></details>

The first one is a bit ugly because the text alignment is off.
The second one isn't supported yet.

Do we want to go with a list anyways?

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/google/mobly/426)*

**Comment by xpconanfan** [comment]
Anything else we can do here?

**Comment by winterfroststrom** [comment]
Nothing really comes to mind for this.

File: README.md [hunk 1]
```diff
@@ -32,6 +28,10 @@ Mobly tests could run on the following platforms:
   - MacOS 10.6+
   - Windows 7+
 
+| Linux | Windows |
```
Comment (xpconanfan) [inline README.md]: Can we make it look like:

Platform    Build Status
Linux            [badge]
Windows      [badge]

Comment (winterfroststrom) [reply README.md]: changed

**Pull request merged**